src/crypto/signature.js

```diff
@@ -9,49 +9,89 @@
 import publicKey from './public_key';
 import enums from '../enums';
 import util from '../util';
+import crypto from '.';
```

Comment:
Do you need this import?

Reply:
nope, left over from an intermediate commit